### PR TITLE
Redesign stage1

### PR DESCRIPTION
**Pull request by catalinaadam**

### CODE REVIEW

File: theme/mail.html [hunk 1]
```diff
@@ -1,98 +1,81 @@
...
+              <noscript>
+                <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-K48FGP2" height="0" width="0" style="display:none;visibility:hidden"></iframe>
+              </noscript>
+              {% endblock %}
+
+              {% block content %}
+
+                <!-- Edit and view button -->
+                {% if page.edit_url %}
+                  {% set edit = "https://github.com/percona/pmm-doc/edit/main/docs/" %}
```
Comment (alina-derkach-oaza) [inline theme/mail.html:41]: Change to {% set edit = "https://github.com/percona-platform/portal-doc/edit/main/docs/" %}

File: theme/mail.html [hunk 2]
```diff
@@ -1,98 +1,81 @@
...
+                  {% set view = "https://raw.githubusercontent.com/percona/pmm-doc/main/docs/" %}
```
Comment (alina-derkach-oaza) [inline theme/mail.html:42]: Change to {% set view = "https://raw.githubusercontent.com/percona-platform/portal-doc/main/docs/" %}

File: mkdocs.yml [hunk 3]
```diff
@@ -68,18 +76,24 @@ markdown_extensions:
...
-      emoji_generator: !!python/name:materialx.emoji.to_svg
+    lightgallery: {}
+    pymdownx.inlinehilite: {}
+    pymdownx.emoji:
+        emoji_index: !!python/name:materialx.emoji.twemoji
+        emoji_generator: !!python/name:materialx.emoji.to_svg
+    - attr_list
+    - pymdownx.emoji:
+        emoji_index: !!python/name:materialx.emoji.twemoji
+        emoji_generator: !!python/name:materialx.emoji.to_svg
```
Comment (alina-derkach-oaza) [inline mkdocs.yml:87]: The following part is redundant. Please remove it. 

    - attr_list
    - pymdownx.emoji:
        emoji_index: !!python/name:materialx.emoji.twemoji
        emoji_generator: !!python/name:materialx.emoji.to_svg

File: mkdocs.yml [hunk 4]
```diff
@@ -68,18 +76,19 @@ markdown_extensions:
...
-  # - bootstrap-tables
-  - section-index # Adds links to nodes - comment out when creating PDF
-  #  - htmlproofer # Uncomment to check links - but extends build time significantly
+  - search
+  section-index: {}
+  search: 
+    separator: '[\s\-,:!=\[\]()"`/]+|\.(?!\d)|&[lg]t;|(?!\b)(?=[A-Z][a-z])'
   - macros:
       include_yaml:
 #        - 'variables.yml' # Use in markdown as '{{ VAR }}'
```
Comment (alina-derkach-oaza) [inline mkdocs.yml:94]: Change the plugins section to match the following

plugins:
  section-index: {}
  search: 
    separator: '[\s\-,:!=\[\]()"`/]+|\.(?!\d)|&[lg]t;|(?!\b)(?=[A-Z][a-z])'
  git-revision-date: {}
  - macros:
      include_yaml:
#        - 'variables.yml' # Use in markdown as '{{ VAR }}'

File: mkdocs.yml [hunk 5]
```diff
@@ -68,18 +76,19 @@ markdown_extensions:
...
-  - attr_list
-  - pymdownx.emoji:
-      emoji_index: !!python/name:materialx.emoji.twemoji
-      emoji_generator: !!python/name:materialx.emoji.to_svg
-
+    lightgallery: {}
+    pymdownx.inlinehilite: {}
+    pymdownx.emoji:
+        emoji_index: !!python/name:materialx.emoji.twemoji
+        emoji_generator: !!python/name:materialx.emoji.to_svg
```
Comment (alina-derkach-oaza) [inline mkdocs.yml:83]: Change the markdown_extensions section to match the following. Mind the spaces

```
markdown_extensions:
  attr_list: {}
  toc:
    permalink: True
  admonition: {}
  footnotes: {}
  def_list: {} # https://michelf.ca/projects/php-markdown/extra/#def-list
  meta: {}
  smarty:
      smart_angled_quotes: true
  pymdownx.mark: {}
  pymdownx.smartsymbols: {}
  pymdownx.tabbed: {}
  pymdownx.tilde: {}
  pymdownx.superfences: {}
  pymdownx.details: {}
  pymdownx.highlight:
      linenums: false
  pymdownx.inlinehilite: {}
  pymdownx.emoji:
      emoji_index: !!python/name:materialx.emoji.twemoji
      emoji_generator: !!python/name:materialx.emoji.to_svg
```

**Review by alina-derkach-oaza — DISMISSED** [review]

**Review by zstefanyshyn — COMMENTED** [review]
Looks good to me!

**Pull request merged**